src/components/academies/AcademyContentCard.jsx

```diff
@@ -29,53 +31,34 @@ const AcademyContentCard = ({
 
   useEffect(
     () => {
-      function contentIntersect(academyContent, tagContent) {
-        const intersect = [];
-        tagContent.forEach((content) => {
-          if (academyContent.some(o => o.aggregation_key === content.aggregation_key)) {
-            intersect.push(content);
-          }
-        });
-        return intersect;
-      }
       async function fetchCourses() {
         setIsAlgoliaLoading(true);
-
-        const { hits: academyHits, nbHits: nbAcademyHits } = await courseIndex.search('', {
-          filters: `(content_type:course OR content_type:learnerpathway) AND academy_uuids:${academyUUID}`, // eslint-disable-line object-shorthand
+        const searchFacetFilters = selectedTag ? [
+          ['content_type:course', 'content_type:learnerpathway'],
+          `academy_uuids:${academyUUID}`,
+          `enterprise_customer_uuids:${enterpriseCustomer.uuid}`,
+          `academy_tags:${selectedTag}`,
+        ] : [
+          ['content_type:course', 'content_type:learnerpathway'],
+          `academy_uuids:${academyUUID}`,
+          `enterprise_customer_uuids:${enterpriseCustomer.uuid}`,
```

Comment:
Why you have added `enterprise_customer_uuids:${enterpriseCustomer.uuid}` in facetFilters?, As per my knowledge, customers are not directly associated with academies. Am I missing something?

Reply:
This is the work done for ENT-8658. We are now scoping the results w.r.t. enterprise customer also